plugins/plugins.bzl

```diff
@@ -29,8 +29,9 @@ def proxy_wasm_plugin_rust(**kwargs):
         **kwargs
     )
 
-def proxy_wasm_plugin_cpp(**kwargs):
+def proxy_wasm_plugin_cpp(copts = [], **kwargs):
     proxy_wasm_cc_binary(
+        copts = copts + ["-Werror=return-type"],
```

Comment:
Any reason not to do "-Werror" by itself, which would turn all warnings into errors?

Reply:
There are build relating warnings tied to other dependencies that we don't necessarily want to create an error for. I think "-Werror" is too heavy handed.

Reply:
SGTM, thanks!